.github/workflows/ci.yml

```diff
@@ -144,90 +144,41 @@ jobs:
         if: matrix.test-mode == 'defaults'
         env:
           TEST_STATE_SCHEME: path
-        run: |
-          packages=`go list ./...`
-          for package in $packages; do
-            echo running tests for $package
-            if ! stdbuf -oL gotestsum --format short-verbose --packages="$package" --rerun-fails=2 --no-color=false -- -coverprofile=coverage.txt -covermode=atomic -coverpkg=./...,./go-ethereum/... -timeout 20m -tags=cionly > >(stdbuf -oL tee -a full.log | grep -vE "INFO|seal"); then
-              exit 1
-            fi
-          done
+        run: ${{ github.workspace }}/.github/workflows/gotestsum.sh --tags cionly --timeout 20m --cover --write-full-log
```

Comment:
On this line and on line 153, aren't you setting the "--timeout 20m" to the default value that would be used if you omitted them?

Reply:
Yep, removed the default timeout.